### ISSUE TITLE
Datepicker - today is obscured

### ISSUE THREAD

**Issue by jeremy-holt** [magnusdanielson/au-fluent-ui#3]
![image](https://user-images.githubusercontent.com/2135027/90500403-58a85100-e13a-11ea-94a6-9f7728f80de7.png)

**Comment by magnusdanielson** [comment]
It is actually a bug from Microsoft. I filed it here:
https://github.com/microsoft/fluentui/issues/14583

**Comment by jeremy-holt** [comment]
I'm looking forward to using au-fluent on a new project. Do I need to know React to get the most out of it?

**Comment by magnusdanielson** [comment]
No, you don't need to know React at all. It is good to know about "Data down-Actions up", a pattern used by most Fluent UI React components. It can be used with Aurelia too. My contribution is the au-wrapper npm package that can wrap any React component. I try not to touch the Fluent UI React components at all, just wrap them properly.

I am glad you like it. We use it in production for our customers.

Good luck!
